app/packages/looker-3d/src/overlays/cuboid.tsx

```diff
@@ -21,6 +21,13 @@ export const Cuboid = ({
   color,
   useLegacyCoordinates,
 }: CuboidProps) => {
+  const geo = React.useMemo(
+    () => dimensions && new THREE.BoxGeometry(...dimensions),
+    [dimensions]
+  );
+
+  if (!location || !dimensions) return null;
```

Comment:
Returning here changes the number of hooks, no? Maybe needs a tweak

Reply:
There's only one hook here that I see and that comes before the return statement.

Reply:
Oh only one hook, my mistake